vitalsigns/vitalsigns.d.ts

```diff
@@ -0,0 +1,193 @@
+
+            /**
+             * The name of the monitor containing the field to be constrained.
+             * @type {string}
+             */
+            monitor: string;
+
+            /**
+             * true to negate the outcome of the comparison; false or omitted to use the comparison result.
+             * @type {boolean}
+             */
+            negate?: boolean;
+
+            /**
+             * The value against which the field should be compared.
+             * @type {any}
+             */
+            value: any;
+        }
+
+        /**
+         * Constraint wrapper.
+         * @interface
+         */
+        export interface ConstraintWrapper {
+            equals?: (num: number) => ConstraintWrapper;
+
+            greaterThan?: (num: number) => ConstraintWrapper;
+
+            lessThan?: (num: number) => ConstraintWrapper;
+
+            not?: ConstraintWrapper;
+        }
+
+        /**
+         * Options.
+         * @interface
+         */
+        export interface Options {
+            /**
+             * Number of milliseconds to wait between automatic health checks.
+             * @type {number|boolean}
+             */
+            autoCheck?: number | boolean;
+
+            /**
+             * HTTP response code to send back in the VitalSigns.
+             * @type {number}
+             */
+            httpHealthy?: number;
+
+            /**
+             * HTTP response code to send back in the VitalSigns.
+             * @type {number}
+             */
+            httpUnhealthy?: number;
+        }
+
+        export interface Monitor {
+            /**
+             * Connections.
+             * @type {any}
+             */
+            connections: any;
+        }
+
+        /**
+         * Monitor field.
+         * @interface
+         */
+        export interface MonitorField {
+            /**
+             * Name.
+             * @type {string}
+             */
+            name?: string;
+
+            /**
+             * Units.
+             * @type {string}
+             */
+            units?: string;
+        }
+
+        /**
+         * Report options.
+         * @interface
+         */
+        interface ReportOptions {
+            /**
+             * true to flatten the report object down to a single level by concatenating nested key names; false to keep the default hierarchical format.
+             * @type {boolean}
+             */
+            flatten?: boolean;
+
+            /**
+             * If flatten is true, this string will be used to separate key names when they are concatenated together.
+             * @type {boolean}
+             */
+            separator?: string;
+        }
+    }
+
+    /**
+     * VitalSigns instance.
+     */
+    interface VitalSigns {
+        /**
+         * Constructor.
+         * @constructors
+         * @param {Options} [options] Options.
+         */
+        new (options?: vitalsigns.Options): VitalSigns;
```

Comment:
As written this actually means that *instances* of `VitalSigns` are new-able. You probably want to `declare class VitalSigns` and give it a `constructor(options?: vitalsigns.Options);` which means that there's a *value* named `VitalSigns` that's new-able. You can then make that the default export, so you can `import VitalSigns = require("vitalsigns")` and create a `new VitalSigns()`.